test/fixtures/test-runner/output/test-timeout-flag.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --test-timeout=20
+'use strict';
+const { describe, test } = require('node:test');
+const { setTimeout } = require('node:timers/promises');
+
+describe('--test-timeout is set to 20ms', () => {
+  test('test 1 must fail due to testTimeoutFailure with error test timed out after 20ms', async () => {
+    await setTimeout(1000);
+  })
+  test('test 2  must fail due to testTimeoutFailure with error test timed out after 600ms', { timeout: 600 }, async () => {
+    await setTimeout(1000);
+  })
```

Comment:
It's probably clearer to have a test that would pass without the specific setting, and one that would pass but don't
```suggestion
  test('should timeout', { timeout: 5 }, async () => {
    await setTimeout(10);
  });
  test('should not timeout', { timeout: 50 }, async () => {
    await setTimeout(25);
  });
```

Reply:
Addressed in 07863c51b228f3475b9751e5f9d4d96ccc2237b2